.solhint.json

```diff
@@ -2,7 +2,7 @@
   "extends": "solhint:recommended",
   "plugins": [],
   "rules": {
-    "compiler-version": ["warn", "^0.8.18"],
+    "compiler-version": ["warn", "^0.8.20"],
```

Comment:
I read online that PUSH0 code causes incompatibilities of EVM on Ethereum and other L1s. We should probably read up on the issues. Various suggestions how to address this

Reply:
100% correct. arbitrum as example not supported push0.
https://ethereum.stackexchange.com/questions/150281/invalid-opcode-opcode-0x5f-not-defined
https://developer.arbitrum.io/solidity-support
better not rush with shanghai as default EVM

Reply:
As long as the chain runs the Shanghai upgrade, we are safe:
https://medium.com/coinmonks/push0-opcode-a-significant-update-in-the-latest-solidity-version-0-8-20-ea028668028a

https://docs.soliditylang.org/en/v0.8.20/using-the-compiler.html?ref=zaryabs.com#setting-the-evm-version-to-target